src/components/tree-view/tree-view.tsx

```diff
@@ -87,7 +98,7 @@ export const TreeItem: React.SFC<TreeItemProps> =
                         {item.label}
                     </span>
                 </div>
-                <div className="nested-tree">
+                {item.children && <ul className="nested-tree" role="group">
```

Comment:
Is this change for accessibility (the item.children part)? There is no test for this

Reply:
its to avoid rendering empty `ul` if no children exist. will modify tests accordingly